src/main/java/com/ericsson/ei/services/ISubscriptionService.java

```diff
@@ -44,19 +44,21 @@
      * @return
      * @throws SubscriptionNotFoundException
      */
-    Subscription getSubscription(String name, String user) throws SubscriptionNotFoundException;
+    Subscription getSubscription(String name) throws SubscriptionNotFoundException;
     
     /**
      * 
      * @param subscription
      * @param subscriptionName
+     * @param user
      * @return
      */
     boolean modifySubscription(Subscription subscription, String subscriptionName, String user);
```

Comment:
I don't understand why we need user as input. An existing Subscription should have userName field already. And subscriptionName is the key for the unique subscription.  Why do we need user parameter? It should be removed in my point of view. Or am I missing something?

Reply:
Do we use 'user', username, in more places where its not needed?

Reply:
here we use userName for preventing modify subscription if current user is not owner of this subscription.
only in two places user param was unneeded, so there I removed this param

Reply:
Ok,, and the user value comes from the RestApi authentication mechanism. Then its ok.